notebooks/210-raft-optical-flow/210-raft-optical-flow.ipynb

```diff
@@ -0,0 +1,700 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        if device != "CPU":
</pre><p>do you tested this branch?</p><p>Probably it is accidentally copied from another notebook (where dynamic input shape was replaced with static for inference on GPU device)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1051/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Thanks for the review, I delete it now